corehq/apps/userreports/tasks.py

```diff
@@ -10,7 +10,7 @@
 from dimagi.utils.couch.database import iter_docs
 
 
-@task(queue='background_queue', ignore_result=True)
+@task(queue='background_queue', ignore_result=True, acks_late=True)
```

Comment:
Keep in mind this will only retry the task if the worker crashes mid-execution (not if some other exception is raised, see http://celery.readthedocs.org/en/latest/faq.html#should-i-use-retry-or-acks-late).

Also, some message brokers (like Redis) implement acks_late through a visibility timeout setting which is a certain amount of time after which unacknowledged tasks get restarted, and this can be a problem for long-running tasks.  But this doesn't seem to be a problem for rabbitmq so as long as we don't change the message broker it shouldn't be an issue.